sdk/storage/azure_storage_blob/src/clients/blob_client.rs

```diff
@@ -38,43 +42,130 @@ use std::sync::Arc;
 pub struct BlobClient {
     pub(super) endpoint: Url,
     pub(super) client: GeneratedBlobClient,
+    pub(super) container_name: String,
+    pub(super) blob_name: String,
+}
+
+impl GeneratedBlobClient {
+    fn from_url(
```

Comment:
You may be able to use `#[tracing::new]` here - it *should* work and hide the tracer initialization.

Reply:
You might want to resync with main (which may be painful because of Heath's recent changes). He ran into the same issue and we decided that the public requirement wasn't necessary or appropriate. 

So changing core is probably the right thing to do.

Reply:
@LarryOsterman Following up on this, was the change to remove the public requirement live yet or still living in an unmerged PR? I am rehydrating this after the Core changes and still facing the error:
<img width="1376" height="67" alt="image" src="https://github.com/user-attachments/assets/fee9d2b7-5019-4ef7-a2c6-65787c1158b9" />

Thanks!

Reply:
Unfortunately it appears it's still in a private branch. If you'd like, you can certainly fix this yourself - it's lines 376 and 377 of `tracing_new.rs` in the azure_core_macros package.